data-prepper-plugins/aws-plugin/src/test/java/org/opensearch/dataprepper/plugins/aws/DefaultAwsCredentialsSupplierTest.java

```diff
@@ -62,4 +64,12 @@ void getProvider_calls_getOrCreate_with_Supplier() {
         when(credentialsProviderFactory.providerFromOptions(options)).thenReturn(awsCredentialsProvider);
         assertThat(actualCredentialsSupplier.get(), equalTo(awsCredentialsProvider));
     }
+
+    @Test
+    void getDefaultRegion_returns_default_region() {
```

Comment:
We should have another test for when it is not present (`Optional.empty()` per my earlier comment).